src/main/java/ui/GuiElement.java

```diff
@@ -0,0 +1,61 @@
+package ui;
+
+import backend.resource.TurboIssue;
+import backend.resource.TurboLabel;
+import backend.resource.TurboMilestone;
+import backend.resource.TurboUser;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+public class GuiElement {
+    private final TurboIssue issue;
+    private final List<TurboLabel> labels;
+    private final Optional<TurboMilestone> milestone;
+    private final Optional<TurboUser> assignee;
+
+    public GuiElement(TurboIssue issue,
+                      List<TurboLabel> labels,
+                      Optional<TurboMilestone> milestone,
+                      Optional<TurboUser> assignee) {
+        this.issue = issue;
+        this.labels = labels;
```

Comment:
Maybe this could be given a more specific name, to indicate that it includes labels referenced in events.